app/assets/javascripts/discourse/app/raw-templates/topic-list-header-column.hbr

```diff
@@ -23,4 +23,5 @@
   {{~#if view.isSorting}}
     {{d-icon view.sortIcon}}
   {{/if ~}}
+  {{~plugin-outlet name="topic-list-heading-bottom"  outletArgs=(raw-hash name=view.className bulkSelectEnabled=bulkSelectEnabled)~}}
```

Comment:
do you want to pass `bulkSelectEnabled` only in the hbr template? and the `name` should be `view.name` I think?